platform-includes/debug-symbols-apple/_default.mdx

```diff
@@ -80,6 +80,7 @@ Follow these steps to upload the debug symbols:
 
 1.  Download and install [sentry-cli](/product/cli/installation/).
 2.  Copy the script below into a new _Run Script_ and set your _AUTH_TOKEN_, _ORG_SLUG_, and _PROJECT_SLUG_.
+3.  Set `ENABLE_USER_SCRIPT_SANDBOXING` in your project settings to `NO`.
```

Comment:
Is the user supposed to do this in the sentry UI? If so, it would help to clarify by adding something like "Go to project settings > setting page name and toggle off X". 

If this should be done in code, then I would make this clear by adding it to the code sample script the previous step references.

Reply:
Not code, not sentry, this is Xcode settings. But this is a good point, I will clarify